recipes/postgresql/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "9.5.3" %}
+
+package:
+    name: postgresql
+    version: {{ version }}
+
+source:
+    fn: postgresql-{{ version }}.tar.bz2
+    url: https://ftp.postgresql.org/pub/source/v{{ version }}/postgresql-{{ version }}.tar.bz2
+    sha256: 7385c01dc58acba8d7ac4e6ad42782bd7c0b59272862a3a3d5fe378d4503a0b4
+    patches:
+        - fix_libraries_win.patch  # [win]
```

Comment:
I'm pretty sure I have not submitted it upstream.  It's just a matter of filenames and locations that are expected.  They would likely reject any patch, because they have intentionally set up their build system this way.  We have to do this in a few places - people (us included) do not always do things the same.  We rename static libraries differently, for example.